src/container/srv_cli.c

```diff
@@ -36,15 +36,15 @@ dsc_cont_close(daos_handle_t poh, daos_handle_t coh)
 	if (pool == NULL)
 		D_GOTO(out, rc = -DER_NO_HDL);
 
+	daos_csummer_destroy(&cont->dc_csummer);
```

Comment:
Similarly, moving this call is unnecessary.

We should really comment that the first dc_cont_put call is for the handle (or introduce a dedicated API to make it clear that we are essentially destroying coh, not cont).

Reply:
ok